subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/verification/verifier/DependencyVerifierBuilder.java

```diff
@@ -222,7 +222,7 @@ List<Checksum> buildChecksums() {
         }
 
         public void addTrustedKey(String key) {
-            pgpKeys.add(key);
+            pgpKeys.add(key.toUpperCase());
```

Comment:
❌ Case-changing functions that are not intended to be locale-sensitive should pass `Locale.ROOT` explicitly.